adoptopenjdk-api-v3-frontend/src/test/kotlin/net/adoptopenjdk/api/AssetsResourceReleaseNamePathTest.kt

```diff
@@ -0,0 +1,76 @@
+package net.adoptopenjdk.api
+
+import io.quarkus.test.junit.QuarkusTest
+import io.restassured.RestAssured
+import net.adoptopenjdk.api.v3.dataSources.APIDataStore
+import net.adoptopenjdk.api.v3.dataSources.SortMethod
+import net.adoptopenjdk.api.v3.dataSources.SortOrder
+import net.adoptopenjdk.api.v3.filters.ReleaseFilter
+import net.adoptopenjdk.api.v3.models.ReleaseType
+import net.adoptopenjdk.api.v3.models.Vendor
+import org.junit.jupiter.api.DynamicTest
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestFactory
+import org.junit.jupiter.api.extension.ExtendWith
+import java.util.stream.Stream
+
+@ExtendWith(value = [DbExtension::class])
+@QuarkusTest
+class AssetsResourceReleaseNamePathTest : FrontendTest() {
```

Comment:
We should add a test for the a successful request (HTTP 200 response) as well that shows what a valid response looks like.

Reply:
ok, sure